### PR TITLE
Version in popup

### PR DESCRIPTION
**Pull request by GrahamSH-LLK**
Adds version in popup:
image
Links to releases page, although this could be changed.

### CODE REVIEW

**Comment by Explosion-Scratch** [comment]
Nice! That's sort fo useful I guess.

**Comment by Explosion-Scratch** [comment]
Maybe having it in the alt text would be better though.

**Comment by WorldLanguages** [comment]
It looks like this for me
image

File: webpages/popup/index.html [hunk 1]
```diff
@@ -11,6 +11,7 @@
       <div id="title">
         <img src="../../images/icon.svg" id="logo" alt="Logo" />
         <span id="title-text">Scratch Addons</span>
+        <span><a id="version" href="https://scratchaddons.com/changelog">...</a></span>
```
Comment (WorldLanguages) [inline webpages/popup/index.html:14]: Why specify a `href` if you'll then trap the click event and open a different page?

Comment (GrahamSH-LLK) [reply webpages/popup/index.html:14]: Because I forgot to remove that. 😄

**Comment by Hans5958** [comment]
Don't it is better to have white, or, if you want, white with 90% of transparency? Also it is kind of misaligned.

**Comment by Hans5958** [comment]
Also, a suggestion.

https://github.com/ScratchAddons/ScratchAddons/blob/5a1fb4d7da9dfe93c02a93297c534fcaebac27fe/webpages/popup/index.js#L72

How about do this?

```js
window.open(`https://github.com/ScratchAddons/ScratchAddons/releases/tag/v${chrome.runtime.getManifest().version}`);
```

That will link the version's release page directly, instead of linking the releases tab.

For example:
- https://github.com/ScratchAddons/ScratchAddons/releases/tag/v1.2.0 (this won't work because it is not released yet)
- https://github.com/ScratchAddons/ScratchAddons/releases/tag/v1.1.1 (for version 1.1.1)

**Comment by GrahamSH-LLK** [comment]
> Also, a suggestion.
> 
> https://github.com/ScratchAddons/ScratchAddons/blob/5a1fb4d7da9dfe93c02a93297c534fcaebac27fe/webpages/popup/index.js#L72
> 
> How about do this?
> 
> ```js
> window.open(`https://github.com/ScratchAddons/ScratchAddons/releases/tag/v${chrome.runtime.getManifest().version}`);
> ```
> 
> That will link the version's release page directly, instead of linking the releases tab.
> 
> For example:
> 
> * https://github.com/ScratchAddons/ScratchAddons/releases/tag/v1.2.0 (this won't work because it is not released yet)
> * https://github.com/ScratchAddons/ScratchAddons/releases/tag/v1.1.1 (for version 1.1.1)

I _was_ going to do that, but I didn't want there to ever be 404s. The general releases page always has the most recent release on the top.

**Comment by Hans5958** [comment]
That would be not possible, if done right. The page creation, which happens when a commit is tagged (not released), happens before the future version submitted to the Chrome Web Store (and AMO, but you are a Chrome dude), so there will be any 404 pages, unless (a) WL tagged it with a different tag (like `1.2.0`, `v.1.2.0` `v1.2`), (b) WL forgot to tag the commit, or (c) the version is on development, so it would be useless to check the release page.

Also, why do you want to link the latest release, and not the release page of the current installed version? Remember that there would be people who wanted to install older versions, both GitHub and AMO

----

Also, you forgot this message.

> Isn't it better to have white, or, if you want, white with 90% of transparency? Also it is kind of misaligned.

**Comment by GrahamSH-LLK** [comment]
> Isn't it better to have white, or, if you want, white with 90% of transparency? Also it is kind of misaligned.

Sorry for not responding. Yes, it does seem to be a bit misaligned. I'll fix that. I like gray, but if you want white, I'll play with that a bit.

**Comment by GrahamSH-LLK** [comment]
True... What if it fetches the link, sees if it's a 404, and if it's a 404, it sends you to the general releases. Otherwise it does what you said.

**Comment by WorldLanguages** [comment]
@GrahamSH-LLK I think that might not be possible because of CORS (and we don't have host permissions for github.com)

**Comment by Hans5958** [comment]
> Sorry for not responding. Yes, it does seem to be a bit misaligned. I'll fix that. I like gray, but if you want white, I'll play with that a bit.
I think that make sense if you do it on a white background (because transparent black + white = gray). I'd rather going to an approach of making it white with 80-90% transparent, just so it blends with the orange background.

> True... What if it fetches the link, sees if it's a 404, and if it's a 404, it sends you to the general releases. Otherwise it does what you said.
We shouldn't not over-complicate this. Also, this is something that won't be clicked so often, so don't worry about it.

**Comment by WorldLanguages** [comment]
Can I merge this?

**Pull request merged**